internal/util-cache/src/main/scala/sbt/internal/util/AdditionalFormats.scala

```diff
@@ -0,0 +1,57 @@
+package sbt.internal.util
+
+import sbt.internal.util.Types.:+:
+
+import sjsonnew.{ Builder, deserializationError, JsonFormat, Unbuilder }
+import sjsonnew.BasicJsonProtocol, BasicJsonProtocol.asSingleton
+
+import java.io.{ ByteArrayInputStream, ByteArrayOutputStream, InputStream, OutputStream }
+
+trait SetFormat { self: BasicJsonProtocol =>
```

Comment:
Set is also here.
https://github.com/eed3si9n/sjson-new/blob/master/core/src/main/scala/sjsonnew/CollectionFormats.scala#L136